code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+			continue
+		else if (origin == M)
+			//to_chat(world, "Target [M] skipped; [M] is origin: [origin].") //DEBUG
+			continue
+		else if (M in existing_targets)
+			//to_chat(world, "Target [M] skipped; [M] is in existing targets.") //DEBUG
+			continue
+		else
+			possible_targets += M
+			//to_chat(world, "Target [M] added.") //DEBUG
+	if(!possible_targets.len)
+		return
+	var/mob/living/next = pick(possible_targets)
+	//to_chat(world, "Target [next]. Possible targets: [possible_targets]. Bolt Energy [bolt_energy]. User: [user]") //DEBUG
+	var/new_damage = bolt_energy - bolt_decay_per_bounce
+	if(next && new_damage >= 1)
+		Bolt(current,next,new_damage,bounces - 1,bolt_range - 1, user)
+
+#undef FORCELIGHTNING_REFLECT_MULTIPLIER
+
+// SITH LORD ENDS
+
+// JUSTICIAR'S FOLLOWER BEGINS
+
+// hey, this one is called traitor_chaplain. might as well, right? -SgtHunk
+
+// Traitor Kit. 20TC, Chaplain only.
+
+/obj/item/storage/box/syndicate/bundle_clock/PopulateContents()
+	new /obj/item/clothing/suit/space/clockwork(src)
+	new /obj/item/clothing/gloves/combat/clockwork(src)
+	new /obj/item/clothing/shoes/combat/swat/bronze(src)
+	new /obj/item/clothing/glasses/hud/clock(src)
+	new /obj/item/clothing/head/helmet/space/clockwork(src)
+	new /obj/item/book/granter/martial/justiciar_hand(src)
+
+// Whole set of Armor.
+
+/obj/item/clothing/head/helmet/space/clockwork
+	name = "brass hardsuit helmet"
+	desc = "Ancient helmet made out of brass. It seems to provide robust protection against most damage types. It glows with a golden, ancient and forgotten energy."
+	worn_icon = 'icons/Fulpicons/clockchap.dmi'
+	icon = 'icons/obj/clothing/clockwork_garb.dmi'
+	icon_state = "clockwork_helmet"
+	inhand_icon_state = "clockwork_helmet"
+	armor = list("melee" = 50, "bullet" = 50, "laser" = 20, "energy" = 20, "bomb" = 30, "bio" = 100, "rad" = 30, "fire" = 100, "acid" = 100)
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	actions_types = list()
+
+/obj/item/clothing/gloves/combat/clockwork
+	name = "brass gauntlets"
+	desc = "Strong gauntlets, that offer protection from weaponry at ease."
+	worn_icon = 'icons/Fulpicons/clockchap.dmi'
+	icon = 'icons/obj/clothing/clockwork_garb.dmi'
+	icon_state = "clockwork_gauntlets"
+	inhand_icon_state = "clockwork_gauntlets"
+	clothing_flags = NOSLIP
+	armor = list("melee" = 15, "bullet" = 15, "laser" = 10, "energy" = 10, "bomb" = 20, "bio" = 10, "rad" = 0, "fire" = 80, "acid" = 50)
+
+/obj/item/clothing/shoes/combat/swat/bronze
+	name = "brass boots"
+	desc = "A giant pair of shoes made out of enchanted bronze."
+	worn_icon = 'icons/Fulpicons/clockchap.dmi'
+	icon = 'icons/obj/clothing/clockwork_garb.dmi'
+	icon_state = "clockwork_treads"
+	armor = list("melee" = 15, "bullet" = 15, "laser" = 10, "energy" = 10, "bomb" = 50, "bio" = 10, "rad" = 0, "fire" = 70, "acid" = 50)
+
+/obj/item/clothing/glasses/hud/clock
```

Comment:
cool thermals

too bad neither they nor the helmet give you flash protection, so you'll NEED to either grab a welding gas mask (or wait ages for welding eyes) or wear sunglasses instead of these if you don't want to get dunked on

I mean, I guess you could switch between these and sunnies mid-combat, but IMO, I think it'd be simpler to just give the helmet of this armor set flash protection